project_7.html

```diff
@@ -393,22 +388,22 @@ <h3 class="mb-3" id="pj-heading-tm1637">TM1637 7-Segment Display</h3>
 					composed of seven
 					LEDs arranged into segments (as shown in the image below).
 				</p>
-				<img class="img-fluid mb-4" src=".\assets\images\project_7\ops_seven_segment_display.webp">
-				<p class="mb-3">
+				<img class="pj-img-fluid mx-auto mb-4" src=".\assets\images\project_7\ops_seven_segment_display.webp">
+				<p class="mb-5">
```

Comment:
I realized that if almost every `<p>` is an `mb-5` maybe we should just write a CSS rule so that we don't have to make each individual bootstrap change?